tools/pycbc_test_suite.sh

```diff
@@ -61,16 +61,18 @@ if [ "$PYCBC_TEST_TYPE" = "search" ] || [ -z ${PYCBC_TEST_TYPE+x} ]; then
     popd
 
     # run PyCBC Live test
-    pushd examples/live
-    bash -e run.sh
-    if test $? -ne 0 ; then
-        RESULT=1
-        echo -e "    FAILED!"
-        echo -e "---------------------------------------------------------"
-    else
-        echo -e "    Pass."
+    if ((${PYTHON_MINOR_VERSION} > 7)); then
```

Comment:
Could you add a FIXME here to remind us that this is needed because of ligo.skymap and python-ligo-lw?